templates/zerver/help/manage-your-uploaded-files.md

```diff
@@ -1,19 +1,64 @@
 # Manage your uploaded files
 
+Zulip lets you view, sort, filter, download, and delete any files that you have
+uploaded.
+
+## View a file
+
 {start_tabs}
 
 {settings_tab|uploaded-files}
 
+1. Click on the name of a file in the **File** column.
+
+!!! tip ""
+
+    You can also view a file in the context of the conversations where it
+    was mentioned by clicking on a message ID in the **Mentioned in** column.
+
 {end_tabs}
 
-From there, you can
+## Sort your files
+
+You can sort your uploaded files by name, upload date, message ID, and size.
 
-* **View all your files**
-* **Delete files**
-* **Download files**
+{start_tabs}
 
-and more.
+{settings_tab|uploaded-files}
+
+1. Click on the name of a column to toggle between ascending and descending order.
+
+!!! tip ""
+
+    You can filter your current view by using the **search box** near the top
+    right corner of the app.
+
+{end_tabs}
+
+## Download a file
+{start_tabs}
+
+{settings_tab|uploaded-files}
+
+1. In the **Actions** column, click the **download file**
+   (<i class="fa fa-download"></i>) icon for the file you want to download.
+
+{end_tabs}
+
+## Delete a file
+
+{start_tabs}
+
+{settings_tab|uploaded-files}
+
+1. In the **Actions** column, click the **delete file**
+   (<i class="fa fa-trash-o"></i>) icon for the file you want to remove.
+
+1. Approve by clicking **Delete**.
+
```

Comment:
Let's explain what happens to the message when you delete a file. Maybe something like:

If you delete a file, your message will still contain a file link, but the link will no longer work.